README.md

```diff
@@ -170,6 +109,24 @@ INSERT INTO users_roles VALUES (ID, 1);
 
 Repeat the steps to get a new access_token.
 
+Check endpoints :
+
+- Check Mapathon Summary :
+
+```
+curl -d '{"project_ids": [11224, 10042, 9906, 1381, 11203, 10681, 8055, 8732, 11193, 7305,11210, 10985, 10988, 11190, 6658, 5644, 10913, 6495, 4229],"fromTimestamp":"2021-08-27T9:00:00","toTimestamp":"2021-08-27T11:00:00","hashtags": ["mapandchathour2021"]}' -H 'Content-Type: application/json' http://127.0.0.1:8000/v1/mapathon/summary/
+```
+  It should return some stats
+
+- Check Mapathon detailed report :
+  You can test  with the `/mapathon/detail/` endpoint  with the following input to check both authentication and database connection
+
+```
+{"fromTimestamp":"2019-04-08 10:00:00.000000","toTimestamp":"2019-04-08 11:00:00.000000","projectIds":[1],"hashtags":[]}
```

Comment:
I have empty result : `{"totalContributors":0,"mappedFeatures":[]}`

When I see the table public.osm_changeset in tests/src/features/insights.sql, I only see 2018 changesets and no 2021 changesets.

Reply:
That is expected result , Try with the Mapathon summary you should see results , You can build your own query by looking at the data available or I can build on request body based on the data !

Reply:
That is expected result , Try with the Mapathon summary you should see results , You can build your own query by looking at the data available or I can build on request body based on the data !

Reply:
In fact I returned you the result of the curl on mapathon summary you set in the README, I commented the bad line ...

It is better to have a non empty result with the data in the repository.

We can also remove it from the README and use the integrations tests to validade that the full setup is ok.

Reply:
Oh okay ! I have added mapathon summary.sql file to insert in insight ,i have modified doc to use this too , its in test fixtures . If you insert it it should return endpoint with data and that's good idea it is better to remove null response , I will insert relavant data to show case detail endpoint

Reply:
And this will be temporary , we are very close to provide underpass dump regularly once it will be available devs people can play with it using api

Reply:
I have updated doc with the relevant request body that will return the result for mapathon/detailed endpoint 
[c9b9b7b](https://github.com/hotosm/galaxy-api/pull/335/commits/c9b9b7b948f6ee857bb9ca378780b0ec2886daff)